src/VisualStudio/Core/Def/Implementation/ProjectSystem/RuleSets/VisualStudioRuleSetManager.cs

```diff
@@ -50,5 +50,10 @@ public void Dispose()
 
             _ruleSetFileMap.Clear();
         }
+
+        void IDisposable.Dispose()
```

Comment:
Maybe add a comment that it only implements `IDisposable` so it can be used with a `using` statement in tests.